packages/web/src/scss/components/Navigation/index.html

```diff
@@ -0,0 +1,88 @@
+{{#> web/layout/default title="Navigation" parentPageName="Components" }}
+
+<section class="UNSTABLE_Section">
+
+  <div class="Container">
+
+    <h2 class="docs-Heading">Navigation</h2>
+
+    <div class="docs-Stack docs-Stack--stretch">
+
+      <div style="--spirit-header-height: 72px;">
```

Comment:
Shouldn't the height be provided by `Header`? For standalone usage, we could have a reasonable fallback in the `var()` function.

Reply:
Solved by adding a fallback and renaming the CSS variable and setting it to the Navigation (with NavigationLink) directly.